workmanager/force-app/main/default/aura/WIContactCard/WIContactCardController.js

```diff
@@ -0,0 +1,32 @@
+({
+    workIcon : function(component, event, helper) {
+        var cardType = component.get("v.workItem.Type__c")
+        
+        if(cardType == "Idea"){
+            component.set("v.iconType","action:description");
+        }else if(cardType == "Action"){
+            component.set("v.iconType","action:fallback");
+        }else if(cardType == "Fix"){
+            component.set("v.iconType","action:canvas");
+        }else{
+            return null;
+        }
+        
```

Comment:
The logic should be inside the helper, as a best practice in order to accomplish more reusable code.

Reply:
Will update it, thanks!